### PR TITLE
Added tests for changing admin/upgrading protocol via governance

### PR DESCRIPTION
**Pull request by jferas**
Added test for changing admin via governance

Add Drips protocol upgrade via governance tests

### CODE REVIEW

**Comment by jferas** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/1?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #2** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/2?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #5** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/5?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #8** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/8?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #12** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/12?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #15** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/15?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #14** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/14?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #19** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/19?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #18** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/18?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                * **PR #21** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/21?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
            * **PR #13** <a href="https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/13?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ScopeLift/radworks-governor-upgrade/21?utm_source=stack-comment).

File: test/RadworksDripsGovernance.t.sol [hunk 1]
```diff
@@ -118,6 +164,34 @@ abstract contract RadworksDripsGovernance is ProposalTest {
     // Ensure the admin role has been renounced
     assertEq(drips.admin(), address(0));
   }
+
+  function test_proposeNewAdminViaGovernance(address _newAdmin) public {
```
Comment (alexkeating) [inline test/RadworksDripsGovernance.t.sol:168]: Name should be `testFuzz_ProposaNewAdminViaGovernance`

File: test/RadworksDripsGovernance.t.sol [hunk 2]
```diff
@@ -2,8 +2,25 @@
 pragma solidity ^0.8.20;
 
 import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {UUPSUpgradeable} from "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
 import {ProposalTest} from "test/helpers/ProposalTest.sol";
 
+import {console2} from "forge-std/console2.sol";
```
Comment (alexkeating) [inline test/RadworksDripsGovernance.t.sol:8]: Leftover `console2`

File: test/RadworksDripsGovernance.t.sol [hunk 3]
```diff
@@ -2,8 +2,25 @@
...
+
+/// @dev This contract used in the testing of using governance to upgrade the Drips protocol
```
Comment (alexkeating) [inline test/RadworksDripsGovernance.t.sol:10]: "*is* used in the testing of *governance* to upgrade the Drips protocol*.*" this sounds a little better

**Review by alexkeating — APPROVED** [review]
Looks good, two small changes left but should be good to merge after

**Review by wildmolasses — APPROVED** [review]
lgtm after alex's suggestions

File: test/RadworksDripsGovernance.t.sol (hunk 1 above)
Comment (jferas) [reply test/RadworksDripsGovernance.t.sol]: Done: https://github.com/ScopeLift/radworks-governor-upgrade/pull/21/commits/ec11571ab8c1595c882aab4119eb39672a50a00a

File: test/RadworksDripsGovernance.t.sol (hunk 2 above)
Comment (jferas) [reply test/RadworksDripsGovernance.t.sol]: Done: [8706cb8](https://github.com/ScopeLift/radworks-governor-upgrade/pull/21/commits/8706cb80f190e2fe306252f46ba610e3abc85de4)

File: test/RadworksDripsGovernance.t.sol (hunk 3 above)
Comment (jferas) [reply test/RadworksDripsGovernance.t.sol]: Done: https://github.com/ScopeLift/radworks-governor-upgrade/pull/21/commits/79efaeab8b3dbcbd6b2bb47c70c5ffd7cbe123ed

**Pull request merged**